cmd/orchestrate/orchestrate.go

```diff
@@ -0,0 +1,296 @@
+// The orchestrate service plays the role of processing messages
+// in stand-alone operations.
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	uuid "github.com/google/uuid"
+
+	"sda-pipeline/internal/broker"
+	"sda-pipeline/internal/config"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	queueVerify    string = "verified"
+	queueInbox     string = "inbox"
+	queueComplete  string = "completed"
+	queueBackup    string = "backup"
+	queueMapping   string = "mappings"
+	queueIngest    string = "ingest"
+	queueAccession string = "accessionIDs"
+)
+
+type upload struct {
+	Operation          string      `json:"operation"`
+	User               string      `json:"user"`
+	Filepath           string      `json:"filepath"`
+	Filesize           int         `json:"filesize"`
+	LastModified       string      `json:"file_last_modified,omitempty"`
+	EncryptedChecksums []checksums `json:"encrypted_checksums,omitempty"`
+}
+
+type request struct {
+	User               string      `json:"user"`
+	Filepath           string      `json:"filepath"`
+	DecryptedChecksums []checksums `json:"decrypted_checksums"`
+}
+
+type trigger struct {
+	Type               string      `json:"type"`
+	User               string      `json:"user"`
+	Filepath           string      `json:"filepath"`
+	EncryptedChecksums []checksums `json:"encrypted_checksums"`
+}
+
+// finalize struct that holds the json message data
```

Comment:
```suggestion
```

Seems like a leftover that should be removed.

Reply:
Sorted.